tests_perf/tests/series.py

```diff
@@ -1,7 +1,7 @@
 import hpat
 
 from .common import Implementation as Impl
-from .data_generator import StringSeriesGenerator, WhiteSpaceStringSeriesGenerator
+from .data_generator import StringSeriesGenerator, WhiteSpaceStringSeriesGenerator, FloatSeriesGenerator, FloatSeriesIndexGenerator
```

Comment:
Please shorten the line:
from .data_generator import (StringSeriesGenerator, WhiteSpaceStringSeriesGenerator,
                             FloatSeriesGenerator, FloatSeriesIndexGenerator)